dockers/docker-fpm/config.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sonic-cfggen -m /etc/sonic/minigraph.xml -t /etc/swss/bgp/bgpd.conf.j2 >/etc/quagga/bgpd.conf
+sonic-cfggen -m /etc/sonic/minigraph.xml -t /etc/swss/bgp/zebra.conf.j2 >etc/quagga/zebra.conf
+
```

Comment:
yes, but if return non zero, what will your config.sh do? goes on?

---
In reply to: [96070348](https://github.com/Azure/sonic-buildimage/pull/190#discussion_r96070348) [](ancestors = 96070348)